### ISSUE TITLE
when the train converge, the loss is ...?

### ISSUE THREAD

**Issue by SophieChang66** [hysts/pytorch_mpiigaze#5]

**Comment by hysts** [comment]
Sorry, but I don't understand the question. Could you tell me a bit more specific?
